cmd/list/accessrequests/cmd.go

```diff
@@ -0,0 +1,117 @@
+package accessrequests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/accesstransparency/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use   = "accessrequest"
+	short = "List access requests"
+	long  = "List access requests in Pending or Approved status. " +
+		"If '--cluster' flag is used, list all access requests in any status for the specified cluster."
+	example = `  # List all access requests for cluster 'foo'
+  rosa list accessrequest --cluster foo
+  `
+)
+
+func NewListAccessRequestsCommand() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: []string{"access-request", "accessrequests", "access-requests"},
+		Short:   short,
+		Long:    long,
+		Example: example,
+		Run:     rosa.DefaultRunner(rosa.RuntimeWithOCM(), ListAccessRequestsRunner()),
+		Args:    cobra.NoArgs,
+	}
+
+	output.AddFlag(cmd)
+	ocm.AddOptionalClusterFlag(cmd)
+	return cmd
+}
+
+func ListAccessRequestsRunner() rosa.CommandRunner {
+	return func(_ context.Context, r *rosa.Runtime, cmd *cobra.Command, _ []string) error {
+		clusterId := ""
+		if cmd.Flags().Changed("cluster") {
+			cluster, err := r.OCMClient.GetCluster(r.GetClusterKey(), r.Creator)
+			if err != nil {
+				return err
+			}
+			clusterId = cluster.ID()
+		}
+		accessRequests, err := r.OCMClient.ListAccessRequest(clusterId)
+		if err != nil {
+			return err
+		}
+		if output.HasFlag() {
+			output.Print(accessRequests)
+		} else {
+			if len(accessRequests) == 0 {
+				if clusterId == "" {
+					r.Reporter.Infof("There are no Access Requests in Pending or Approved status.")
+				} else {
+					r.Reporter.Infof("There are no Access Requests for cluster '%s'.", r.ClusterKey)
+				}
+				return nil
+			}
+			writer := tabwriter.NewWriter(os.Stdout, 0, 4, 2, ' ', 0)
+			output, hasPending, pendingId := getAccessRequestsOutput(clusterId, accessRequests)
+			fmt.Fprint(writer, output)
+			err = writer.Flush()
+			if err != nil {
+				return err
+			}
+			if hasPending {
+				r.Reporter.Infof("Run the following command to approve or deny the access request:\n\n"+
+					"   rosa create decision --access-request %s --decision Approved\n"+
```

Comment:
Since the flag for creating a decision using a `-` between `access` and `request` - I think it makes sense (along with other naming convention in ROSA) to change the default for this command to `access-request` rather than `accessrequest`
WDYT?